apps/kol-amelamdim/pages/api/upload-file.ts

```diff
@@ -29,14 +30,13 @@ export default async function handler(req, res) {
 
     if (formData?.files?.sharedFile && selectedCategory) {
       // read file from the temporary path
-      const fileSize = formData.files.sharedFile.size * 1e-6;
+      const fileSize = megaBytes(formData.files.sharedFile.size);
```

Comment:
need some verb in the name of the function

```suggestion
      const fileSize = formatBytes(formData.files.sharedFile.size);
```

Reply:
Resolved.